src/main/java/com/google/flourbot/entity/EntityModule.java

```diff
@@ -0,0 +1,86 @@
+package com.google.flourbot.entity;
+
+import com.google.flourbot.entity.EntityInterface;
+import com.google.flourbot.datastorage.FirebaseDataStorage;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.flourbot.entity.action.Action;
+import com.google.flourbot.entity.action.SheetAppendAction;
+import com.google.flourbot.entity.trigger.Trigger;
+import com.google.flourbot.entity.trigger.CommandTrigger;
+import java.util.Map;
+
+import java.lang.Exception;
+
+public class EntityModule implements EntityInterface {
+    
+    private static EntityModule instance = null;
+    private final FirebaseDataStorage datastorage;
+
+    private EntityModule() {
+        this.datastorage = new FirebaseDataStorage();
```

Comment:
please read at the bottom how I propose to do it... just wanted to point out explicitly that this type of initialization is risky. DataStorage is supposed to be a singleton, and you have no guarantee that another module will construct it again.

Reply:
Will address in Issue #11